src/OctoshiftCLI.Tests/gei/Handlers/MigrateRepoCommandHandlerTests.cs

```diff
@@ -325,10 +328,35 @@ public async Task Happy_Path_GithubSource_Ghes()
 
             var gitArchiveUrl = $"https://example.com/{gitArchiveId}";
             var metadataArchiveUrl = $"https://example.com/{metadataArchiveId}";
-            var gitArchiveContent = new byte[] { 1, 2, 3, 4, 5 };
-            var metadataArchiveContent = new byte[] { 6, 7, 8, 9, 10 };
+
             var authenticatedGitArchiveUrl = new Uri($"https://example.com/{gitArchiveId}/authenticated");
             var authenticatedMetadataArchiveUrl = new Uri($"https://example.com/{metadataArchiveId}/authenticated");
+            var gitArchiveFilePath = "/";
+            var metadataArchiveFilePath = "/";
+
+            var gitArchiveTempPath = System.IO.Path.GetTempPath() + "integration_test_archive";
+
+            if (File.Exists(gitArchiveTempPath))
+            {
+                File.Delete(gitArchiveTempPath);
+            }
+
+            using var gitArchiveContent = File.Create(gitArchiveTempPath);
```

Comment:
We don't want to create a file here as I mentioned before since this is a unit test. everything need to be tested with abstractions. Since we have already tests the `HttpDownlaodService.DownlaodFile()` method we can just mock it and maybe verify that we're calling it.

Reply:
Addressed in 56688e760716150fa9c4414d0f43c58a1ee74343 :tada: